diff --git a/ports/unix/mpconfigport.mk b/ports/unix/mpconfigport.mk
index f0aa955c0..c5857593f 100644
--- a/ports/unix/mpconfigport.mk
+++ b/ports/unix/mpconfigport.mk
@@ -28,11 +28,11 @@ MICROPY_PY_USSL = 1
 # axTLS has minimal size and fully integrated with MicroPython, but
 # implements only a subset of modern TLS functionality, so may have
 # problems with some servers.
-MICROPY_SSL_AXTLS = 1
+MICROPY_SSL_AXTLS = 0
 # mbedTLS is more up to date and complete implementation, but also
 # more bloated. Configuring and building of mbedTLS should be done
 # outside of MicroPython, it can just link with mbedTLS library.
-MICROPY_SSL_MBEDTLS = 0
+MICROPY_SSL_MBEDTLS = 1
 
 # jni module requires JVM/JNI
 MICROPY_PY_JNI = 0

